main/src/modules/attentionRelated/ears/src/ears.cpp

```diff
@@ -26,6 +28,36 @@ bool ears::configure(yarp::os::ResourceFinder &rf)
     MainGrammar = rf.findFileByName(rf.check("MainGrammar", Value("MainGrammar.xml")).toString());
     bShouldListen = true;
 
+    if (followPlans){
+        /*portToBehavior.open("/" + moduleName + "/behavior:o");
+        while (!Network::connect(portToBehavior.getName(),"/GoalManager/trigger:i ")) {
+            yWarning() << " Behavior is not reachable";
+            yarp::os::Time::delay(0.5);
+        }*/
+
+        // port to planner module
+        port_planner.open("/" + moduleName + "/target:o");
+        while (!Network::connect(port_planner.getName(),"/planner/rpc"))
+        {
+            yWarning() << "Planner is unreachable...";
+            yarp::os::Time::delay(0.5);
+        }
+
+        goal_need_port.open("/" + moduleName + "/goal:o");
+    }else{
+        portToBehavior.open("/" + moduleName + "/behavior:o");
+        while (!Network::connect(portToBehavior.getName(),"/BehaviorManager/trigger:i ")) {
+            yWarning() << " Behavior is not reachable";
+            yarp::os::Time::delay(0.5);
+        }
+
+        portTarget.open("/" + moduleName + "/target:o");
```

Comment:
This is already opened in Line 25 (https://github.com/robotology/wysiwyd/blob/13d8816fbaa506f3637f31ebe7bf22b05961c041/main/src/modules/attentionRelated/ears/src/ears.cpp#L25)